src/Symfony/Component/Validator/Constraints/Url.php

```diff
@@ -31,7 +31,7 @@ class Url extends Constraint
     ];
 
     public string $message = 'This value is not a valid URL.';
-    public string $tldMessage = 'This URL does not contain a TLD.';
+    public string $tldMessage = 'This URL is missing a domain extension.';
```

Comment:
alternative: `This URL is missing a top-level domain.`
better?

Reply:
Yes, this is more precise and can be searched on internet if someone doesn't understand the message.

Reply:
+1 for `This URL is missing a top-level domain.`

I think the important thing is that the developer knows what the error message means, so that a) they can support the end-user b) they can adapt the text to their own context.

Reply:
PR updated!